src/neuroconv/tools/nwb_helpers.py

```diff
@@ -184,3 +189,25 @@ def make_or_load_nwbfile(
 
                 if not success and not file_initially_exists:
                     nwbfile_path_in.unlink()
+
+
+def get_io_datasets(nwbfile):
+    for object_ in nwbfile.objects():
+        if isinstance(object_, TimeSeries):
+            for field in ("data", "timestamps"):
+                if getattr(object_, field) is not None and not isinstance(getattr(object_, field), DataIO):
+                    yield object_.id, field
+        if isinstance(object_, DynamicTable):
+            for field in getattr(object_, "colnames"):
```

Comment:
Out of curiosity, what is this condition for?
`isinstance(getattr(object_, field), DataIO)`

Is this so we don't overwrite previous configuration? 

Also, what happens in this function if the object was previously bound as in we read this from a file already and we are appending new datasets? Is it related to this?